data_safe_haven/exceptions/__init__.py

```diff
@@ -9,7 +9,8 @@ class DataSafeHavenCloudError(DataSafeHavenError):
 class DataSafeHavenConfigError(DataSafeHavenError):
     pass
 
-
+# This is a custom exception class for handling errors when interacting with Entra ID.
+# For example, when adding users to an Entra group fails.
 class DataSafeHavenEntraIDError(DataSafeHavenCloudError):
     pass
```

Comment:
```suggestion
class DataSafeHavenEntraIDError(DataSafeHavenCloudError):
    """
    This is a custom exception class for handling errors when interacting with Entra ID.
    For example, when adding users to an Entra group fails.
    """
    pass
```

If you make this a docstring instead of a comment then it will show up on e.g. tooltips in an IDE.

Reply:
And in API documentation if we ever do that.